packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -752,28 +720,40 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
      * @param indexTo last price request index to update the trackers for.
      */
     function updateTrackersRange(address voterAddress, uint256 indexTo) external {
+        _resolveResolvablePriceRequests(pendingPriceRequestsIds.length);
         require(
-            voterStakes[voterAddress].nextIndexToProcess < indexTo && indexTo <= priceRequestIds.length,
+            voterStakes[voterAddress].nextIndexToProcess < indexTo && indexTo <= resolvedPriceRequestIds.length,
             "Invalid indexTo"
         );
 
         _updateAccountSlashingTrackers(voterAddress, indexTo);
     }
 
+    function resolveResolvablePriceRequests() external {
+        _resolveResolvablePriceRequests(pendingPriceRequestsIds.length);
```

Comment:
nit: this might be a bit _less contextual_ if we just pass in UINT256_MAX here, so it will process as many as it can and leave it to the method to figure out how many that should be. Also might be a bit cheaper.

Reply:
sure. done.